showcase.json

```diff
@@ -517,19 +517,20 @@
   {
     "id": "rubaru",
     "name": "RuBaRu",
-    "oneLiner": "Building Onchain Regenerative Creator-Consumer Economy",
+    "oneLiner": "The fully on-chain content creator/influencer platform: own your profile, own your content, and earn fair incentives.",
```

Comment:
The use of _The_ implies that it aims to be the only one. Maybe this is the intent? Otherwise, _A_ might be more appropriate.